CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
 # ---- CMake options ----
 
-cmake_minimum_required(VERSION 3.5.2 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
 
-# Set cmake policy by version: https://cmake.org/cmake/help/latest/manual/cmake-policies.7.html
+# Set cmake policy by version: https://cmake.org/cmake/help/latest/manual/cmake-
+# policies.7.html
 if(${CMAKE_VERSION} VERSION_LESS 3.12)
```

Comment:
Zero chance that it <3.12 given the instruction above.

Reply:
Correct. I should indeed remove those lines if I impose CMake >= 3.13